dev/namrathagopalabhatla/classification_models.py

```diff
@@ -0,0 +1,148 @@
+import numpy as  np
+import pandas as pd
+
+import matplotlib.pyplot as plt 
+
+from sklearn.neural_network import MLPClassifier
+from sklearn.metrics import accuracy_score, classification_report, confusion_matrix, precision_recall_fscore_support, f1_score
+from sklearn.linear_model import LogisticRegression
+
+from seaborn import heatmap
+from IPython.display import display
+
+class MLP:
+    def __init__(self, hidden_size=None, activation='relu', solver='lbfgs', max_iter=1000):
+        '''Constructor for initalization of model using provided parameters'''
+        super(MLP, self).__init__()
```

Comment:
This (and line 86) is not needed if your class is not inheriting explicitly from another one.